contrib/CMakeLists.txt

```diff
@@ -182,6 +182,28 @@ else()
     message(STATUS "Not using QPL")
 endif ()
 
+if (OS_LINUX AND ARCH_AMD64)
+    option (ENABLE_QATLIB "Enable Intel® QuickAssist Technology (Intel® QATlib)" ${ENABLE_LIBRARIES})
+elseif(ENABLE_QATLIB)
+    message (${RECONFIGURE_MESSAGE_LEVEL} "QAT library is only supported on x86_64 arch")
+endif()
+if (ENABLE_QATLIB)
+    option (ENABLE_USDM_DRV "A User Space DMA-able Memory (USDM) component which allocates/frees DMA-able memory" OFF)
+    option (ENABLE_QAT_OUTOFTREE "Using out-of-tree driver, user needs customize ICP_ROOT variable" OFF)
+    set(ICP_ROOT "" CACHE STRING "ICP_ROOT variable to define the path of out-of-tree driver package")
+if (ENABLE_QAT_OUTOFTREE)
+if (ICP_ROOT STREQUAL "")
```

Comment:
Something is broken with the indentation here.

Reply:
Done